CMakeLists.txt

```diff
@@ -581,6 +581,12 @@ install(
     DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/applications
 )
 
+install(
+    FILES
+    ${CMAKE_SOURCE_DIR}/freeorion.appdata.xml
+    DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/appdata
```

Comment:
You can use either; `/usr/share/metainfo` is the new location but appdata will be scanned basically forever and the old location works on older distros like Fedora 23 -- it's up to you. If this is a new file, I'd go with the new location.